fabric-fluids-v1/src/main/java/net/fabricmc/fabric/mixin/fluids/PlayerInventoryMixin.java

```diff
@@ -0,0 +1,68 @@
+package net.fabricmc.fabric.mixin.fluids;
+
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.player.PlayerInventory;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.collection.DefaultedList;
+import net.minecraft.world.World;
+
+import net.fabricmc.fabric.api.fluids.items.ItemSink;
+
+@Mixin (PlayerInventory.class)
+public abstract class PlayerInventoryMixin implements ItemSink {
```

Comment:
Why does this need to mixin PlayerInventory

Reply:
I thought it would be convenient to have this, so right clicking tanks and managing fluid interactions with the player would be easier